README.md

```diff
@@ -99,8 +99,32 @@ in use when sequence metadata file was created.
 ```python
 >>> from cladetime import CladeTime
 
-# Create a CladeTime object for any date after May, 2023
->>> ct = CladeTime(sequence_as_of="2024-10-15")
+# Create a CladeTime object for recent historical data
+>>> ct = CladeTime(sequence_as_of="2025-10-15")
+```
+
+### Data Availability Constraints
+
+> [!IMPORTANT]
+> **Historical data availability is limited** due to Nextstrain's data retention policies:
+>
+> - **Sequence data**: Available from **2025-09-29** onwards (approximately 7 weeks of history)
```

Comment:
There are a few references in here to a Nextstrain "policy" of keeping "~7 weeks" of data. But that is not really accurate, as I'm not sure there is a policy per se. Maybe we could just manually change these references to being a bit more vague ("at least 90 days" or something like that?).